zend_abstract_interface/sandbox/sandbox.h

```diff
@@ -218,8 +220,25 @@ inline bool zai_sandbox_timed_out(void) {
     return false;
 }
 
+inline bool zai_is_request_blocked(void)
```

Comment:
This is missing the redirection message as well.

Reply:
Done